workflow/controller/controller.go

```diff
@@ -307,7 +307,7 @@ func (wfc *WorkflowController) startLeading(ctx context.Context, logCtx *log.Ent
 	go wfc.workflowGarbageCollector(ctx.Done())
 	go wfc.archivedWorkflowGarbageCollector(ctx.Done())
 
-	go wfc.runTTLController(ctx, workflowTTLWorkers)
+	go wfc.runGCcontroller(ctx, workflowTTLWorkers)
```

Comment:
maybe just call this func `run`

Reply:
Will have seperate names for each queue.